package.json

```diff
@@ -202,12 +202,13 @@
     "html-webpack-plugin": "4.5.0",
     "husky": "4.3.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "26.6.0",
-    "jest-circus": "^26.6.3",
-    "jest-junit": "12.0.0",
+    "jest": "^28.1.0",
```

Comment:
We have been using pinned dependencies for Jest and some of the other packages, is there any reason we can't continue doing that?